lib/api/2.0/nodes.js

```diff
@@ -79,10 +79,14 @@ var nodesDelTagById = controller(function(req) {
 });
 
 var nodesPatchTagById = controller(function(req) {
-    return nodes.addTagsById(req.swagger.params.identifier.value, 
+    return nodes.addTagsById(req.swagger.params.identifier.value,
                              req.swagger.params.body.value.tags);
```

Comment:
@uppalk1 You can simplify this to req.body.tags since the body parameter has generic object schema.